packages/typescriptlang-org/src/components/layout/SiteFooter.tsx

```diff
@@ -82,10 +81,6 @@ const useTypeScriptLinks = [
     title: "TSConfig Ref",
     url: "/tsconfig",
   },
-  {
-    title: "Code Samples",
-    url: "/play/#show-examples",
-  },
```

Comment:
Can we just remove the popover and leave it as a link here? The actual playground page has a version of this.

Reply:
Sure, added it back.